client/src/shared/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+
+interface SearchBarProps {
```

Comment:
- Export this interface as per Sir Joshua's instruction for all shared components.
- You can use this in the future

```suggestion
export interface SearchBarProps {
```